lib/util.rb

```diff
@@ -20,6 +20,14 @@ def self.rootish_ssh(host, user, keys, cmd)
     end
   end
 
+  def self.create_root_certificate(ubid:, duration:)
```

Comment:
Instead of ubid, getting common_name would be more appropriate, we would want to pass urls in some cases for example. Or even subject, but that seems too long. Probably CN provides better balance.

Reply:
updated as requested